### PR TITLE
Copy editing Patricia Merkle Tries page

### PR DESCRIPTION
**Pull request by TimDaub**
- superseeds #8727

I edited the text to make it more readable. I didn't make any changes to the actual content, just copyediting.

### CODE REVIEW

**Comment by minimalsm** [comment]
Thanks for the PR @TimDaub 👍 

@jmcook1186 any chance you could a look at this when you've got some time. no rush :)

File: src/content/developers/docs/data-structures-and-encoding/patricia-merkle-trie/index.md [hunk 1]
```diff
@@ -61,30 +62,32 @@ The update and delete operations for radix tries are simple, and can be defined
...
 
 ## Merkle Patricia Trie {#merkle-patricia-trees}
 
-However, radix tries have one major limitation: they are inefficient. If you want to store just one (path,value) binding where the path is (in the case of the ethereum state trie), 64 characters long (number of nibbles in `bytes32`), you will need over a kilobyte of extra space to store one level per character, and each lookup or delete will take the full 64 steps. The Patricia trie introduced here solves this issue.
+But radix tries have one major limitation: they are inefficient. If you want to store one `(path, value)` binding where the path, like in Ethereum, is 64 characters long (the number of nibbles in `bytes32`), we will need over a kilobyte of extra space to store one level per character, and each lookup or delete will take the full 64 steps. The Patricia trie introduced in the following solves this issue.
```
Comment (jmcook1186) [inline src/content/developers/docs/data-structures-and-encoding/patricia-merkle-trie/index.md:73]:
```suggestion
Radix tries have one major limitation: they are inefficient. If you want to store one `(path, value)` binding where the path, like in Ethereum, is 64 characters long (the number of nibbles in `bytes32`), we will need over a kilobyte of extra space to store one level per character, and each lookup or delete will take the full 64 steps. The Patricia trie introduced in the following solves this issue.
```

**Review by jmcook1186 — CHANGES_REQUESTED** [review]
Thanks - this page needed refining - just one tiny suggestion, but otherwise approve.

File: src/content/developers/docs/data-structures-and-encoding/patricia-merkle-trie/index.md [hunk 2]
```diff
@@ -61,30 +62,32 @@ The update and delete operations for radix tries are simple, and can be defined
...
+
+It is impossible for an attacker to provide a proof of a `(path, value)` pair that does not exist since the root hash is ultimately based on all hashes below it. Any underlying modification would change the root hash.
 
-While traversing a path one nibble at a time, as described above, most nodes contain a 17-element array. One index for each possible value held by the next hex character (nibble) in the path, and one to hold the final target value if the path has been fully traversed. These 17-element array nodes are called `branch` nodes.
+We'll refer to an atomic unit of a radix tree (e.g. a single hex character, or 4 bit binary number) as a "nibble".  While traversing a path one nibble at a time, as described above, nodes can maximally refer to 16 children but include a `value` element. We, hence, represent them as array with length . We call these 17-element arrays "branch nodes".
```
Comment (corwintines) [inline src/content/developers/docs/data-structures-and-encoding/patricia-merkle-trie/index.md:69]:
```suggestion
We'll refer to an atomic unit of a radix tree (e.g. a single hex character, or 4 bit binary number) as a "nibble".  While traversing a path one nibble at a time, as described above, nodes can maximally refer to 16 children but include a `value` element. We, hence, represent them as array with length. We call these 17-element arrays "branch nodes".
```

**Review by corwintines — APPROVED** [review]
Thanks @TimDaub!

**Pull request merged**